### PR TITLE
Add IPA building instructions

### PR DESCRIPTION
**Pull request by fmuyassarov**
Add custom IPA ramdisk image building instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,49 @@
...
+    - `output` - Output base file name
+    - `centos` - Base distribution.
+    - [extra-hardware](https://docs.openstack.org/ironic-python-agentbuilder/latest/admin/dib.html#ironic-python-agent-ipa-extra-hardware)
+    DIB element is required to install required utilities for improving introspection.
+    - [dynamic-login](https://docs.openstack.org/diskimage-builder/latest/elements/dynamic-login/README.html)
+    DIB element allows to inject SSH key in the image.
+
+At the end of the process you will have 
+- `ironic-python-agent.initramfs` - deploy ramdisk.
+-  `ironic-python-agent.kernel` - a binary file containing the kernel.
```
Comment (kashifest) [inline README.md:49]:
```suggestion
- `ironic-python-agent.kernel` - a binary file containing the kernel.
```

**Review by kashifest — APPROVED** [review]
Thanks a lot @fmuyassarov for adding this.

**Comment by dtantsur** [comment]
/approve

Nice! I assume you confirmed that IPA-builder works in a venv?

**Comment by fmuyassarov** [comment]
> /approve
> 
> Nice! I assume you confirmed that IPA-builder works in a venv?

Yep it does.

**Pull request merged**